core-common/src/main/java/org/uniprot/core/util/EnumDisplay.java

```diff
@@ -32,29 +41,45 @@
     }
 
     /**
-     * ignore case and trim on passed string argument to match
+     * Returns the Enum constant by its name. String value of name is trimmed and case is ignored
```

Comment:
not by name its name
by whatever is returning from getCompareOn() method

Reply:
It is talking about the parameter. I have changed it to 
Return the Enum constant by passed name ....